### DISCUSSION TITLE
💖 looking for molecule ec2 driver maintainers

### DISCUSSION THREAD

**Discussion by ssbarnea** [ansible-community/molecule-plugins#202] [ec2]
I am unable to maintain this driver as I am not using it, so I am looking for maintainers. 

I can keep providing support for CI and integration with molecule itself but not around driver specifics.

**Comment by Shaps** [comment]
I’m going to need this driver soon, so happy to work on it as best as I can!

**Comment by ghost** [comment]
i can also work.

**Comment by ghost** [comment]
we are already using it very heavily, so will be happy to contribute.

**Comment by jgoldschrafe** [comment]
Count me in as well.

**Comment by ssbarnea** [comment]
I am still looking for people, please ping me on irc if you want to take it under your wing.

**Comment by tbugfinder** [comment]
Made it any of the volunteers into the maintainers?

**Comment by ssbarnea** [comment]
Invitation is still open and the fact the the project has 5 open PRs is not very good.

I can help only when pull-requests are passing CI and anyone can contact me (zbr) on freenode #ansible-lint channel.

Shortly, the invitation is still open.

**Comment by ssbarnea** [comment]
@gundalow Please restore my rights, so I can give others triage and later write access.

**Comment by luupux** [comment]
@ssbarnea sorry I wroted using wrong accont (infra-agilelab) so my offer still valid but on my personal account @luupux

**Comment by ssbarnea** [comment]
I invited everyone involved in this thread to be a collaborator, initially with triage but I will boost it as soon I get some extra confirmation.

**Comment by luupux** [comment]
sorry to bother with boring question but in this case what it's mean collaboration with triage ?

**Comment by ssbarnea** [comment]
Mainly it means you can open/close/edit issues. Once you do few reviews I will enable write access, which means ability to merge.

There is only one older pull-request open which seems abandoned by OP, still, take a look at it and see if you can extract something useful out of it before we close it.

I guess that once done, you will want me to help with a new release. Ping me and I will tag a new one if one of you can confirm that master code works fine with current release of molecule/ansible.

**Comment by jgoldschrafe** [comment]
Hey @ssbarnea, sorry to bother, but I completely neglected to ever ack the invite, but could have sworn I did. Would you mind sending another one?

**Comment by ssbarnea** [comment]
@jgoldschrafe Somehow I missed the reply, still interested?  I sent the invite again.

I observed that this repo lacked active maintenance for a very long time and I will have to drop it from the list of maintained plugins if nobody steps in to keep at least the CI green and compatible with newer versions of molecule.

**Comment by ssbarnea** [comment]
Anyone still interested in helping with maintenance of this plugin? I need some some help with review from people using this in production, especially as some proposed PRs have the potential of introducing regressions.

**Comment by beargiles** [comment]
I hope to submit several PRs soon and could probably help with the maintenance afterwards. (It will give you a chance to evaluate my skills, etc.)  We're just starting to use molecule + ec2 at work so I wouldn't say this is a 'production' work - but I am using it to develop and maintain both my ansible scripts that manage dev and qa resources.
